src/llmcompressor/utils/helpers.py

```diff
@@ -1082,3 +1083,6 @@ def patch_attr(base: object, attr: str, value: Any):
             setattr(base, attr, original_value)
         else:
             delattr(base, attr)
+
+
+DISABLE_QAC_MODIFIERS = ["GPTQModifier", "AWQModifier", "AutoRoundModifier"]
```

Comment:
just an fyi, this just wont run Quant-Dequant during any forward passes done through https://github.com/vllm-project/compressed-tensors/blob/52792be02ec09e59f3517104e755a02d0e003fbb/src/compressed_tensors/quantization/lifecycle/forward.py#L366